File: pgm/ABC_C#/ABC_142_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int N;
        N = int.Parse(arr[0]);
        Console.WriteLine(Math.Floor((decimal)(N + 1) / 2) / N);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_142_A.vb
' Imports System.Collections.Generic
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N As Integer = Console.ReadLine()
        Console.WriteLine(Math.Floor((N + 1) / 2) / N)
        Return
    End Sub
End Class
